modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/formatActionURL.ts

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
@dsanz Here we have how param interpolation work in actions and the [test](https://github.com/liferay-frontend/liferay-portal/pull/3983/files#diff-5e63b40daa8691b123e457c9992ae97929c00f0b540784c1b6a7e1fca08210c5R37) 
Also tested for the [QuickActions](https://github.com/liferay-frontend/liferay-portal/pull/3983/files#diff-8429be49e45d79a7dd5f5c31048b72465a49f6ac6f3927b81c481a8502f4e9a9R74)